README.md

```diff
@@ -1,104 +1,168 @@
+> # ... more entries ...`
+> ```
+> `⇨` denotes a tab character.
+> 
+> Given the following `PAC-ID`:
+> ```
+> HTTPS://PAC.METTORIUS.COM/DEVICE/21:210263
+> ```
+> the first row of the mapping table would resolve to the following URL:
+> ```
+> https://www.mettorius.com/inventory/DEVICE/210263
+> ```
+> the second would resolve to this URL:
+> ```
+> https://attributes.mettorius.com/DEVICE/21:210263
+> ```
+
+#### Variables
+
+For **Template Url** and **Applicable If** colums, the `variable`s outlined below MAY be used.
+
+> [!NOTE]
+> The following `PAC-ID` (with two [T-REX](https://github.com/ApiniLabs/T-REX) extensions) is used as example for the **Example** column below:
+> ```
+> HTTPS://PAC.METTORIUS.COM/DEVICE/21:210263*11$T.D:20231121+FOO$T.A:BAR*CAL$T.D:20231211
+> ```
+
+| **Variable** | **Description** | **Example** |
+| :--- | :--- | :--- |
+| {isu} | The `issuer` of the `PAC-ID` | {isu} → METTORIUS.COM |
+| {pac} | The complete `PAC-ID` in URL representation (without extensions) | {pac} → HTTPS://PAC.METTORIUS.COM/DEVICE/21:210263 |
+| {id} | The `identifier` of the `PAC-ID` | {id} → DEVICE/21:210263 |
+| {idSegN} | The Nth `id segment` of the `PAC-ID` | {idSeg2} → 21:210263 |
+| {idValFOO} | The `id segment value` of the first `id segment` with `id segment key` "FOO".[^1]  | {idVal21} → 210263 |
+| {ext} | All extensions separated by `*`. | {ext} → 11$T.D:20231121+FOO$T.A:BAR*CAL$T.D:20231211 |
+| {extN} | The Nth extension. | {ext1} → 11$T.D:20231121+FOO$T.A:BAR |
+| {extNSegM} | The Mth segment of the Nth extension. A `+` character separates an extension into segments. | {ext1Seg1} → 11$T.D:20231121 |
+| {extNValFOO} | The value of the extension segment with key "FOO"[^1]  of the Nth extension. Key and value inside a segment are separated by the `:` character. | {ext1Val11$T.D} → 20231121 |
 
-**A:** If a link to your system requires authentication, normal authentication takes place always, independent of how you navigate to this link.
+[^1]: Any occurrence of curly brackets (`{` and `}`) inside "FOO" need to be escaped with a backslash `\` (resulting in `\{` and `\}`).
 
-**Q: What if a mapping URL is changing?**
+#### Rules
 
-**A:** Update the mapping table. Use a local override, like a user or corporate mapping table if needed.
+In the context of this specification, a `rule` consists of a `variable` followed by an `=` sign and a `value`, serving for comparison against a specific value the `variable` refers to inside the `PAC-ID`. Alternatively, if a value for the `variable` merely needs to exist without specifying an exact value, the `variable` can stand alone without an assigned value.
 
-**Q: How to use only a part of the id or category in placeholders?**
+> [!NOTE]
+> Example of a `rule` matching `PAC-ID`s with `issuer` "METTORIUS.COM" and having any non-empty `id segment value` available for the `id segment key` "21":
```

Comment:
This is a check of key existence not of the value being non-empty of that key. at least not explicitly